internal/pproflogging/pproflogging.go

```diff
@@ -63,23 +63,29 @@ const (
 	ProfileNameCPU = "cpu"
 )
 
-// ProfileConfig configuration flags for a profile.
-type ProfileConfig struct {
-	flags []string
-	buf   *bytes.Buffer
+var (
+	// ErrEmptyProfileName returned when a profile configuration flag has no argument.
+	ErrEmptyProfileName = errors.New("empty profile flag")
+
+	//nolint:gochecknoglobals
+	pprofConfigs = newProfileConfigs(os.Stderr)
+)
+
+// Writer interface supports destination for PEM output.
+type Writer interface {
```

Comment:
Writer is a special case.  In most cases, writer will be `*io.File`, but also want to allow more generic types for testing.  `WriteString` support is necessary for proper log output.